prisma/setup.md

```diff
@@ -19,11 +19,19 @@ Starting `psql` on WSL 2 is done with `sudo service postgresql start`
 
 ### Setup Database
 
-Assuming we want to use a database called `bresearchdev` we create it by logging into `psql` with `postgres` user. Then create a password using `\p` and follow the prompts. Memorize the password for the DB connection string. Create the database with `\c bresearchdev`.
+Assuming we want to use a database called `bresearchdev`.
+We log into postgres using `psql postgres`.
+
+Then find out your username using `\du` and create a password using `\password <YourUserName>` and follow the prompts. Memorize the password for the DB connection string.
+
+Create the database with `CREATE DATABASE bresearchdev;`
+Connect to the database with `\c bresearchdev`.
+
+_You can double check your current databases using `\l`_
 
 ### Environment Variables
 
-Create a `.env` (`.env.production` should contain the production AWS url). Set `DATABASE_URL="postgresql://postgres:<YOUR PASSWORD HERE>@localhost:5432/bresearchdev?schema=public"`. Prisma should now be able to seed data using `npx prisma db seed` and `npx prisma studio` should reflect those changes.
+Create a `.env` (`.env.production` should contain the production AWS url). Set `DATABASE_URL="<YOUR POSTGRES USERNAME>://postgres:<YOUR PASSWORD HERE>@localhost:5432/bresearchdev?schema=public"`
```

Comment:
This should be `DATABASE_URL=postgresql://<YOUR POSTGRES USERNAME>:<YOUR PASSWORD HERE>